### PR TITLE
Brew results

### PR DESCRIPTION
**Pull request by BenzGitHub**
Added brewery results for Story One

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -161,9 +161,10 @@ <h3>Filter Options</h3>
 
 </script>
 
-<script src="./assets/javascript/secrets/secrets.js"></script>
```
Comment (Sarah-Git) [inline index.html]: What is "secrets.js" it should be assets/javascript/app.js from my understanding

Comment (BenzGitHub) [reply index.html]: Jordan is using the secrets link to hide the firebase info.
I have added the app.js link in the html.

Comment (Sarah-Git) [reply index.html]: Great job!

**Comment by Sarah-Git** [comment]
MERGE IT

**Pull request merged**